als-suggestions/shared/src/test/resources/test/oas30/by-directory/yaml/open-api-document/security/security-openId.yaml

```diff
@@ -0,0 +1,9 @@
+openapi: 3.0.0
+components:
+  securitySchemes:
+    aScheme:
+      type: openIdConnect
+      openIdConnectUrl: https://example.com/.well-known/openid-configuration
```

Comment:
add test with openIdConnect and also scopes?

Reply:
Added